kubespawner/spawner.py

```diff
@@ -2779,3 +2810,22 @@ async def _ensure_namespace(self):
                 # It's fine if it already exists
                 self.log.exception("Failed to create namespace %s", self.namespace)
                 raise
+
+    async def delete_forever(self):
+        """Called when a user is deleted.
+
+        This can do things like request removal of resources such as persistent storage.
+        Only called on stopped spawners, and is likely the last action ever taken for the user.
+
+        This will only be called once on the user's default Spawner.
+        Supported by JupyterHub 1.4.0+.
```

Comment:
```suggestion
        Supported by JupyterHub 1.4.1+.
```

@minrk besides not being sure about if 1.4.1 or 1.4.0 is required, I consider this PR ready for merge.

Reply:
Sure, let's say 1.4.1. It's added in 1.4, but has some bugs that mean it's probably good to be clear not to expect it to work before 1.4.1